client/src/modules/users/user.modal.html

```diff
@@ -12,16 +12,28 @@
   </ol>
 </div>
 
-  <div class="modal-body">
+  <div class="modal-body" style="overflow-y: scroll; max-height:600px; ">
+
 
     <div class="form-group" ng-class="{ 'has-error' : UserForm.$submitted && UserForm.display_name.$invalid }">
-      <label class="control-label" translate>FORM.LABELS.USERNAME</label>
-      <input name="display_name" ng-model="UserModalCtrl.user.display_name" autocomplete="off" ng-maxlength="UserModalCtrl.maxLength" class="form-control" required>
-      <div class="help-block" ng-messages="UserForm.display_name.$error" ng-show="UserForm.$submitted">
-        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      <label class="control-label col-md-3" translate>FORM.LABELS.USERNAME</label>
+      <div class="col-md-12">
```

Comment:
Having a `col-md-3` and `col-md-12` on the same line doesn't make any sense ... bootstrap only has 12 columns.  Which do you mean to put here - `col-md-3` and `col-md-9`?

Reply:
the curent disposition of the form is : the input is underneath the lable that is why it should have this class 'col-md-12', I'm change the label class col-md-3 to col-md-12 too.
 The 'col-md-12' class in the username input help to set the seach icon visible

Reply:
It doesn't work without the `col-md-12`?  That's interesting.

Reply:
It works but the seach icon position is bad